azure-cli.js

```diff
@@ -28,6 +56,19 @@ async function execute({ command, credentials }) {
   }
 }
 
+async function createVolumeConfigsWithPathMatch(command) {
+  const pathMatches = kaholoPluginLibrary.helpers.extractPathsFromCommand(command);
```

Comment:
Why `pathMatches`? This function is meant to return just `paths`.
If `match` refers to regexps, then it's an implementation detail, which we shouldn't reveal (unless there's a good reason to do so). That detail propagates in the function stack, and becomes confusing to the understanding of the mechanism at high abstract level (when reading `execute` in this case).

Details in the names are important if they matter in the scope, like:
-`pathsArray`
-`customersWithAddressInformation`
they're not, if they're unrelevant to the scope:
-`pathsMatchesArray`
-`customersWithAddressInformationObtainedFromDB`

however if we performed the tests of various path extracting functions then:
-`pathMatchesArray`
-`pathsObtainedWithSomeCoolAlgorithmArray`
would make perfect sense.

Lemme know if you have any questions.